index.html

```diff
@@ -413,6 +413,26 @@ <h3>
       </li>
     </ul>
   </p>
+  <p>
+    The <dfn>contributing factors change threshold exceeded</dfn> steps given |factors| and |state|, are as follows:
+    <ol>
+      <li>
+        If |factors| exceed an [=implementation-defined=] threshold for |state|, return true.
+      </li>
+      <li>
+        Return false.
+      </li>
+    </ol>
+    <aside class="note">
+      The [=contributing factors change threshold exceeded=] algorithm allows user agents to avoid
+      flip-flopping between two states in certain circumstances. This specification does not define
```

Comment:
maybe clarify "certian"

Reply:
I expanded this further with an abstract example.